sql/updates/world/6.x/9999_99_99_17997_world.sql

```diff
@@ -0,0 +1,39 @@
+CREATE TABLE `areatrigger_template` (
+  `id` int(11) unsigned NOT NULL,
+  `flags` int(11) unsigned NOT NULL DEFAULT '0',
+  `moveCurveId` int(11) unsigned NOT NULL DEFAULT '0',
+  `scaleCurveId` int(11) unsigned NOT NULL DEFAULT '0',
+  `morphCurveId` int(11) unsigned NOT NULL DEFAULT '0',
+  `facingCurveId` int(11) unsigned NOT NULL DEFAULT '0',
+  `data0` float NOT NULL DEFAULT '0',
+  `data1` float NOT NULL DEFAULT '0',
+  `data2` float NOT NULL DEFAULT '0',
+  `data3` float NOT NULL DEFAULT '0',
+  `data4` float NOT NULL DEFAULT '0',
+  `data5` float NOT NULL DEFAULT '0',
+  `scriptName` char(64) NOT NULL DEFAULT '',
+  PRIMARY KEY (`id`)
+) ENGINE=MyISAM DEFAULT CHARSET=utf8;
```

Comment:
UpperCamelCase style for columns since all new fields are named so
default uint32 type is int(10) unsigned
add a VerifiedBuild field for tables that have sniffable content